packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -35,7 +38,7 @@ type Opts = {|
 |};
 
 export default class AssetGraphBuilder extends EventEmitter {
-  graph: AssetGraph;
+  graph: OldAssetGraph;
```

Comment:
Will `OldAssetGraph` eventually be deleted and `AssetGraphBuilder` will build two newer graphs -- one of them called `AssetGraph` but will be the newer, smaller one?

Reply:
Yes, that's the plan.